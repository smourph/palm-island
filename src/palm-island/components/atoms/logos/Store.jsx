import React from 'react';
import store from '@internal/resources/images/logo-store.png';
import './Logo.scss';

export const Store = () => (
  <span className="logo logo-action logo-store">
      <img src={store} alt="store"/>
  </span>
);

export default Store;
