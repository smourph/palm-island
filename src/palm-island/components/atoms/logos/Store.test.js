import React from 'react';
import {render, screen} from '@testing-library/react';
import Store from './Store';

it('should render snapshot', () => {
  const {asFragment} = render(<Store/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render store', () => {
  render(<Store/>);

  const icon = screen.getByAltText('store');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-store.png');
});
