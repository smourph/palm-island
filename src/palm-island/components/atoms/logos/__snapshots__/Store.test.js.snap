// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-action logo-store"
>
  <img
    alt="store"
    src="logo-store.png"
  />
</span>
`;
